healthcheck/healthcheck.go

```diff
@@ -15,7 +15,7 @@ import (
 var singleton *HealthCheck
 var once sync.Once
 
-// GetHealthCheck Get HealtchCheck Singleton
+// GetHealthCheck Get HealthCheck Singleton
```

Comment:
Let's have a proper GoDoc string and say something like

`// GetHealthCheck returns the health check which is guaranteed to be a singleton.`